### PR TITLE
Adding Poisson GLM normalization

### PR DESCRIPTION
**Pull request by davidaknowles**

### CODE REVIEW

File: analysis/poisson-glm-normalization.Rmd [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+  title: "Poisson GLM normalization"
```
Comment (jdblischak) [inline analysis/poisson-glm-normalization.Rmd:2]: Please remove the leading whitespace. It causes the following error:

```
Error in yaml::yaml.load(enc2utf8(string), ...) : 
  Parser error: did not find expected <document start> at line 2, column 1
Calls: <Anonymous> ... yaml_load_utf8 -> mark_utf8 -> <Anonymous> -> .Call
Execution halted
```

File: analysis/poisson-glm-normalization.Rmd [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Plot proportions explained by batch vs. individual
+```{r plot-pve-batch-vs-individual}
+cbPalette <- c("#999999", "#E69F00", "#56B4E9", "#009E73", "#F0E442", "#0072B2", "#D55E00", "#CC79A7")
+ggplot( do.call(rbind,batch_over_explained), aes(prop_batch,col=method)) + geom_density(alpha=0.2, position="identity") + xlab("variance_batch / (variance_batch + variance_individual)") + theme(legend.position=c(.8,.8)) + scale_colour_manual(values=cbPalette)+ scale_fill_manual(values=cbPalette)
+```
+
+Plot overall percent variance explained
+```{r plot-pve-total}
+ggplot( do.call(rbind,variance_components), aes(1-residual,col=method)) + geom_density() + xlab("proportion variance explained") + xlim(0,.5)+ scale_colour_manual(values=cbPalette) + theme(legend.position=c(.8,.8))
+```
```
Comment (jdblischak) [inline analysis/poisson-glm-normalization.Rmd:132]: Please add this final section to record session information:

``````
## Session information

```{r info}
sessionInfo()
```
``````

```
```

**Comment by jdblischak** [comment]
Overall looks good! Please address the few minor changes I noted, and then I'll merge the PR. Please let me know if you have any questions.

**Comment by jdblischak** [comment]
I merged the PR and made the small edits. It's now live.

https://jdblischak.github.io/singleCellSeq/analysis/poisson-glm-normalization.html

Thanks!

**Comment by davidaknowles** [comment]
Hi John - thanks for doing this, with traveling and a grant deadline on
Friday it was all a bit hectic. Sorry about that, back on solid ground now.

The good news is that the newest CRAN version of rstan (2.9.0) has the
variational inference functionality exposed again so it would be easy for
you to run the alternative models I coded up. I'll make an .Rmd for that
this week.

Happy New Year! Hope all is going well in Chicago and you're not freezing
_too_ badly.

On 5 January 2016 at 12:22, John Blischak notifications@github.com wrote:

> I merged the PR and made the small edits. It's now live.
> 
> https://jdblischak.github.io/singleCellSeq/analysis/poisson-glm-normalization.html
> 
> Thanks!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jdblischak/singleCellSeq/pull/4#issuecomment-169120997
> .

## 

David A. Knowles,
Stanford University.
E-mail: knowles84@gmail.com
Web: http://cs.stanford.edu/~davidknowles/

**Comment by jdblischak** [comment]
On Mon, Jan 11, 2016 at 12:39 PM, davidaknowles notifications@github.com
wrote:

> Hi John - thanks for doing this, with traveling and a grant deadline on
> Friday it was all a bit hectic. Sorry about that, back on solid ground now.

No problem.

> The good news is that the newest CRAN version of rstan (2.9.0) has the
> variational inference functionality exposed again so it would be easy for
> you to run the alternative models I coded up. I'll make an .Rmd for that
> this week.

Great!

> Happy New Year! Hope all is going well in Chicago and you're not freezing
> _too_ badly.

Thanks! Happy New Year to you as well. We're surviving the cold for now...

**Pull request merged**